src/Model/Photo.php

```diff
@@ -596,6 +596,25 @@ public static function newResource()
 		return System::createGUID(32, false);
 	}
 
+	/**
+	 * Extracts the rid from a local photo URI
+	 *
+	 * @param string $image_uri The URI of the photo
+	 * @return string The rid of the photo, or an empty string if the URI is not local
+	 */
+	public static function ridFromURI($image_uri)
```

Comment:
Plz add the type hint `string`, so `string $image_uri`

Reply:
I don't understand this - isn't this what I already have?  Or do you mean it should read "@param string string $image_uri"?

Reply:
Oh wait, never mind, I get it.  I'm not completely fluent in PHP, or github ;)